common/utils.py

```diff
@@ -0,0 +1,18 @@
+from datetime import date, timedelta
+
+
+class TimeService:
+    """
+    This class handles conversion of date input from users into
+    something python can understand. For instance, when the user types
+    in something like yesterday, we want to return the date of yesterday.
+    """
+
+    def yesterday(self):
+        return date.today() - timedelta(days=1)
+
+    def today(self):
+        return date.today()
+
+    def tomorrow(self):
+        return date.today() + timedelta(days=1)
```

Comment:
Although this looks good, I'm not sure it covers the use case very well. The idea behind the TimeService is this 
if a user puts in `tomorrow`, `yesterday`, `today`, `next tomorrow` we should have a way to parse that information and return the date object for our graphql to use as query filter.

If you look at your implementation, the only difference is the value being passed into `timedelta`. it also takes in a negative number so for where you've got
`date.today() - timedelta(days=1)` is the same as 
`date.today() + timedelta(days=-1)`
and today is same as `date.today() + timedelta(days=0)`

We only need one function
```
def get_date(day_arg=0):
  # Do the magic of conversion inside here
  return date.today() + timedelta(days=day_arg)
```
The TimeService would still need a good way to parse possible entries in order to get the `day_arg`
These are possible different types 
**number_days_type** e.g `5 days, 2 days`
**today_tomorrow_yesterday_type** e.g `today, tomorrow,  `
**day_name_type** e.g `monday , tuesday`
**next_type** e.g `next week, next monday, next wednesday`
**date_type** e.g `12th of March`

We would need another function to accurately determine the `day _arg` based on the type.